client/web/src/components/Auth/SignOut.js

```diff
@@ -40,38 +41,61 @@ const SignOut = ({ isLoggedIn, user, handleDisconnect }) => {
 
   const handleSignout = async () => {
     await signout();
+    console.log('handleSignout function called');
```

Comment:
remove console.log

Reply:
oof , kinda forgot about that , will remove

Reply:
can u remove this